rebar.lock

```diff
@@ -117,5 +119,6 @@
  {<<"regine">>, <<"74A329F769BF5A7DA56C3F04B61AAFFB7276E9AED7D2BF9789F024502CAD0458">>},
  {<<"rfc3339">>, <<"986D7F9BAC6891AA4D5051690058DE4E623245620BBEADA7F239F85C4DF8F23C">>},
  {<<"riak_core">>, <<"712892ECBAAFBA6355452B243E1E9F684CE28585D6D4CC040D1696C98B5B05F7">>},
- {<<"riak_core_lite_util">>, <<"B481D064EBFC3536A51DD412E356B7121CB2FE16B78349B98B65DE73D425E15E">>}]}
+ {<<"riak_core_lite_util">>, <<"B481D064EBFC3536A51DD412E356B7121CB2FE16B78349B98B65DE73D425E15E">>},
+ {<<"setup">>, <<"EFD072578F0CF85BEA96CAAFFC7ADB0992398272522660A136E10567377071C5">>}]}
```

Comment:
Strange, this was already added in https://github.com/travelping/ergw/pull/360/files